index.html

```diff
@@ -5711,6 +5713,28 @@ <h3>Algorithm</h3>
       is used instead of looking at the HTTP Link Header directly. For the sake of simplicity, none of the algorithms
       in this document mention this directly.</p>
 
+    <div class="candidate correction" id="change_5">
+      <span class="marker">Candidate Correction 5</span>
+      <p>Details about transforming to and from a given serialization format
+        have been abstracted so that other specifications can define
+        serialization and deserialization requirements for their specific formats.
+        The algorithm is now more explicit about serializing and deserializing
+        to the <a>internal representation</a>.
+        See marked changes here and in the following locations:</p>
+      <ul>
+        <li><a data-link-for="JsonLdProcessor">compact()</a> <a href="#idl-compact-step-10">step 10</a></li>
+        <li><a data-link-for="JsonLdProcessor">expand()</a> <a href="#idl-expand-step-9">step 9</a></li>
+        <li><a data-link-for="JsonLdProcessor">flatten()</a> <a href="#idl-flatten-step-7">step 7</a></li>
+        <li><a data-link-for="JsonLdProcessor">fromRdf()</a> <a href="#idl-fromRdf-step-3">step 3</a></li>
+        <li><a href="#serialization-and-deserialization" class="sectionRef"></a></li>
+      </ul>
+      <p>For more information, refer to <a href="https://github.com/w3c/json-ld-api/issues/605">issue 605</a>.</p>
+    </div>
+
+    <ins cite="#change_5"><p>Methods resolving a {{Promise}} with the <a>internal representation</a>
+      MUST serialize to JSON (or another appropriate serialization format)
+      as described in <a href="#serialization-and-deserialization" class="sectionRef"></a>.</p></ins>
```

Comment:
```suggestion
    <ins cite="#change_5"><p>Methods resolving a {{Promise}} with the <a>internal representation</a>
      MUST either
      map it to the datatype in used by the implementation, as described in <a href="#internal-representation-in-implementations" class="sectionRef"></a>,
      or serialize it to JSON (or another appropriate serialization format)
      as described in <a href="#serialization-and-deserialization" class="sectionRef"></a>.</p></ins>
```
I think this wording is more aligned with what actual implementations are doing.